backend/src/main/java/org/example/app/task/logic/UcSaveTaskList.java

```diff
@@ -28,11 +28,11 @@ public class UcSaveTaskList {
    * @return the {@link TaskListEntity#getId() primary key} of the saved {@link TaskListEntity}.
    */
   // @RolesAllowed(ApplicationAccessControlConfig.PERMISSION_SAVE_TASK_LIST)
-  public Long save(TaskListEto list) {
+  public TaskListEntity save(TaskListEto list) {
```

Comment:
Returning an Entity from a use case is not good practice. We'll have a talk about this in the training